src/main/java/unibook/commons/core/Messages.java

```diff
@@ -6,7 +6,7 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n";
```

Comment:
You cannot remove `%1$s` from the end of this line. Removing this will cause every other commands error messages to not be displayed. Please add back the `%1$s` part! It's used in add, delete, find, etc.